### PR TITLE
Added cf-net documentation

### PR DESCRIPTION
**Pull request by olehermanse**
Must be merged with https://github.com/cfengine/documentation-generator/pull/203

cf-net is 3.11+ so this is only for 3.11+.

### CODE REVIEW

**Comment by olehermanse** [comment]
@nickanderson @kacf @jimis Please review.

File: reference/components/cf-net.markdown [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+```
+$ sudo /var/cfengine/bin/cf-key
+$ sudo /var/cfengine/bin/cf-agent --bootstrap myhostname
+$ sudo /var/cfengine/bin/cf-net connect
+Connected & authenticated successfully to 'myhostname'
+```
+(`myhostname` can also be an IP address)
+
+All three commands above are run with sudo, so they access the same key file.
+In the future, we would like to make it easier to use cf-net without bootstrapping or sudo.
```
Comment (kacf) [inline reference/components/cf-net.markdown]: I would drop this "in the future..." sentence, statements such as these tend to linger for a long time if we don't end up doing it. I think it belongs in a ticket instead.

File: reference/components/cf-net.markdown [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+## cf-net commands
+`cf-net` usage follows the general structure:
```
Comment (kacf) [inline reference/components/cf-net.markdown]: I would argue *syntax* instead of *usage*.

File: reference/components/cf-net.markdown (hunk 1 above)
Comment (nickanderson) [inline reference/components/cf-net.markdown]: Agreed let's drop that sentence

File: reference/components/cf-net.markdown (hunk 2 above)
Comment (nickanderson) [inline reference/components/cf-net.markdown]: I have no opinion here.

**Comment by olehermanse** [comment]
Followed @kacf's suggestions and fixed both cases. Please merge

**Pull request merged**